### ISSUE TITLE
Update examples to Pixi v3

### ISSUE THREAD

**Issue by dhelbig** [PixelsCommander/HTML-GL#52]
I downloaded the ZIP folder and wanted to run the "filter.html" file from the DEMO folder.
It don't work, because the **Bower_Components** folder with the needed .js files is missing.

`/bower_components/promise-polyfill/Promise.min.js Failed to load 
/bower_components/webcomponents.js/CustomElements.min.js Failed to load 
/bower_components/pixi/bin/pixi.dev.js Failed to load`

Am I doing something wrong?

Best
D.

**Comment by PixelsCommander** [comment]
Hi,  have you installed bower dependencies?

**Comment by dhelbig** [comment]
no, sorry. I did now.

But it seams that pixi package doesn't include a `pixi.dev.js` file anymore.
When I use the regular `pixi.js` file, there appears an error:
`Uncaught TypeError: undefined is not a function(anonymous function)`
refer to this code around line 195 in the filter.html.
`var displacementFilter = new PIXI.DisplacementFilter(displacementTexture);`

I tried to use the `htmlgl.min.js` file (from the PAGE folder) instead of the `pixi.js`, than it works. 
But this all seams really complicated just to get the demo running.
